src/analysis/normed_space/affine_isometry.lean

```diff
@@ -587,3 +588,44 @@ begin
   rw this,
   simp only [homeomorph.comp_is_open_map_iff, homeomorph.comp_is_open_map_iff'],
 end
+
+local attribute [instance, nolint fails_quickly] affine_subspace.nonempty_map
+
+include V₁
+omit V
+
+/--
+An affine subspace is isomorphic to its image under an injective affine map.
+This is the affine version of `submodule.equiv_map_of_injective`.
+-/
+noncomputable def affine_map.equiv_map_of_injective
```

Comment:
```suggestion
noncomputable def affine_subspace.equiv_map_of_injective
```
so that this name matches the convention of the `submodule` version

Reply:
Addressed this and the other suggestions in a commits.